assets/prototype/application/services/toaster/useErrorToast.tsx

```diff
@@ -1,22 +1,27 @@
 import React, { useCallback } from 'react';
 import { ApolloError } from 'apollo-client';
 import { Intent } from '@blueprintjs/core';
+
 import IconGraphQL from './IconGraphQL';
+import { ErrorIcon, ErrorToast, ErrorToastCallback, ToasterMsg } from './types';
 
-const useErrorToast = (toaster, hash) =>
-	useCallback(
+const useErrorToast: ErrorToast = (toaster, hash) =>
+	useCallback<ErrorToastCallback>(
 		(message, timeout = 5000, action = {}, onDismiss = null) => {
-			let icon = 'warning-sign';
-			let errorMessage = message;
+			let icon: ErrorIcon = 'warning-sign';
```

Comment:
Isn't TS inferring type here ?

Reply:
it would infer it from where it's being used in the blueprint Toast component, which would massively expand the number of types it could be. So doing this restricts the options that can be used here.